website/docs/guides/middlewares/consumer-throttling-middleware.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 6
+---
+
+# Consumer Throttling
+
+Consumer Throttling is a feature in KafkaFlow that allows you to dynamically manage the rate at which your application consumes messages from Kafka topics. It provides a mechanism to slow down or pause the consumption of messages based on specified metrics. This can be particularly useful in scenarios where you want to prioritize certain types of messages over others, or when there is a need to control the rate of processing during high-load periods or when dealing with backlogs.
```

Comment:
Add the following as the first line: 
`In this section, we will learn how Consumer Throttling works and how to configure it.`